hexa/databases/graphql/schema.graphql

```diff
@@ -1,6 +1,11 @@
 type Database {
   tables(page:Int, perPage:Int): DatabaseTablePage!
   table(name: String!): DatabaseTable
+  name: String!
+  username: String!
+  host: String!
+  port: Int!
+  externalUrl: String!
```

Comment:
We would need to have the password here as well (otherwise it's not possible to connect to the database). I'd only return the password if the user is an admin of the workspace

Reply:
All right.

Reply:
- `host` should be `slug.explore.openhexa.org`
- `externalUrl` should be replace by `url` and should have the value of the connection string, like the one we use for SQLAlchemy

Reply:
Also, we should use have something like this

```graphql
type Database {
  tables(page:Int, perPage:Int): DatabaseTablePage!
  table(name: String!): DatabaseTable
  credentials: DatabaseCredentials
}

type DatabaseCredentials {
  dbName: String!
  username: String!
  password: String!
  host: String!
  port: Int!
  url: String!
}
```